rclcpp_examples/CMakeLists.txt

```diff
@@ -69,17 +69,17 @@ if(BUILD_TESTING)
     set_and_get_parameters_async
     set_and_get_parameters
     "talker:listener")
-  set(fastrtps_excluded_example_tests
+  set(opensplice_only_example_tests
     "add_two_ints_server:add_two_ints_client"
     "add_two_ints_server:add_two_ints_client_async")
 
   macro(tests)
     set(example_tests_to_test ${example_tests})
     # TODO(wjwwood): recombine with example_tests this when fastrtps supports wait_for_service.
-    if(rmw_implementation STREQUAL "rmw_fastrtps_cpp")
+    if(NOT rmw_implementation STREQUAL "rmw_opensplice")
```

Comment:
The test looks wrong. Have you tried this code locally or on the build farm? There is not rmw implementation called `rmw_opensplice` - only `rmw_opensplice_cpp`.